### DISCUSSION TITLE
Python Client for 3.9 or later Python version

### DISCUSSION THREAD

**Discussion by tanzilkm** [carla-simulator/carla#5571] [Q&A]
Hi,
We are integrating CARLA into a project that is already in progress and uses Python 3.9 or later. Is a Carla client available for the 3.9 version?

Python 3.8 is the latest version that Pypi supports (https://pypi.org/project/carla/).

**Comment by riverallzero** [comment]
Hi @tanzilkm , 
as i know, at the moment there is no wheel available in Pypi which support Python 3.9.

**Comment by ccqil** [comment]
Carla works incorrect with Python 3.9.
As I know, `get_available_maps` and `get_random_location_from_navigation` don't work with Python 3.9.

**Reply by riverallzero** [reply]
Yes, i mean 3.9 is not available
